packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/choice/telephony-line-phone-accessories-choice.service.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Initialize brand list for the filter
+ * @param {*} offers
+ */
+export default function initializeBrandList(offers) {
+  const brandList = ['All'];
+  if (offers.length) {
```

Comment:
If test is useless if you loop in the array. Unless offers may be of different type ?